server/src/test/java/io/spine/server/aggregate/given/AggregateRootTestEnv.java

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright 2017, TeamDev Ltd. All rights reserved.
+ *
+ * Redistribution and use in source and/or binary forms, with or without
+ * modification, must retain the above copyright notice and the following
+ * disclaimer.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+package io.spine.server.aggregate.given;
+
+import io.spine.server.BoundedContext;
+import io.spine.server.aggregate.AggregatePart;
+import io.spine.server.aggregate.AggregatePartRepository;
+import io.spine.server.aggregate.AggregateRoot;
+import io.spine.server.aggregate.Apply;
+import io.spine.server.command.Assign;
+import io.spine.test.aggregate.ProjectDefinition;
+import io.spine.test.aggregate.ProjectDefinitionValidatingBuilder;
+import io.spine.test.aggregate.ProjectId;
+import io.spine.test.aggregate.ProjectLifecycle;
+import io.spine.test.aggregate.ProjectLifecycleValidatingBuilder;
+import io.spine.test.aggregate.Status;
+import io.spine.test.aggregate.command.CreateProject;
+import io.spine.test.aggregate.command.StartProject;
+import io.spine.test.aggregate.event.ProjectCreated;
+import io.spine.test.aggregate.event.ProjectStarted;
+
+public class AggregateRootTestEnv {
+
+    private AggregateRootTestEnv() {
+    }
+
+    public static class AnAggregateRoot extends AggregateRoot<String> {
+        protected AnAggregateRoot(BoundedContext boundedContext, String id) {
+            super(boundedContext, id);
+        }
+    }
+
+    @SuppressWarnings("TypeMayBeWeakened") // Exact message classes without OrBuilder are needed.
```

Comment:
I thought this was just a minor warning according to our inspection profile. If that's still true, I wouldn't bother suppressing it.

We are always going to get warnings like this one, since we use Protobuf messages. In 99% of time those can be weakened to `...MessageOrBuilder`. IDEA uses every chance to remind us about that, making the rest of "TypeMayBeWeakened" warnings invisible for us.